src/greeter/GreeterApp.h

```diff
@@ -38,7 +38,7 @@ namespace SDDM {
     class KeyboardModel;
 
 
-    class GreeterApp : public QGuiApplication
+    class GreeterApp : public QApplication
```

Comment:
Is there a way to "upgrade" a QGuiApplication to a QApplication only as soon as its needed? Or maybe create a separate Process for the MessageBoxes and wait for Termination of that?